File: ejercicio_aulas_ordenadores/ServicioWEB/Ordenador.py
import json

class OrdenadorEncoder(json.JSONEncoder):

    def default(self, obj):
        return obj.__dict__


class Ordenador(object):
    def __init__(self, id_ordenador, aula, modelo, almacenamiento, ram) -> None :
        self.id_ordenador =id_ordenador
        self.aula = aula
        self.modelo = modelo
        self.almacenamiento = almacenamiento
        self.ram = ram

    # def toJSON(self):
    #     #return json.dumps(self, default=lambda o: o.__dict__, sort_keys=True, indent=4)
    #     return json.dump(self)
    #     #return json.dumps(self, skipkeys=False, ensure_ascii=True, check_circular=True, allow_nan=True, cls=None, indent=None, separators=None, default=None, sort_keys=False, **kw)

    def __str__(self) :
        return json.dumps(self)

#http://daniel.blogmatico.com/convertir-un-objeto-python-a-un-objeto-json/

File: ejercicio_aulas_ordenadores/app/src/main/java/com/example/ejercicio_aulas_ordenadores/Modelo/Ordenador.kt
package com.example.ejercicio_aulas_ordenadores.Modelo

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Ordenador(@SerializedName("id_ordenador")
                     val id_ordenador: String? = null,

                     @SerializedName("aula")
                     val aula: String? = null,

                     @SerializedName("modelo")
                     val modelo: String? = null,

                     @SerializedName("almacenamiento")
                     val almacenamiento: String? = null,

                     @SerializedName("ram")
                     val ram: String? = null): Serializable {

}
